packages/client/src/runtime/query.ts

```diff
@@ -1050,10 +1050,13 @@ function byToSelect(by: string[]): Record<string, true> {
   return obj
 }
 
-function getDefaultSelection(outputType: DMMF.OutputType) {
+function getDefaultSelection(dmmf: DMMFHelper, outputType: DMMF.OutputType) {
   const acc = Object.create(null)
 
   for (const f of outputType.fields) {
+    if (dmmf.typeMap[(f.outputType.type as DMMF.OutputType).name] !== undefined) {
+      acc[f.name] = true
+    }
```

Comment:
Maybe add a comment about what this does?

Reply:
Done